### ISSUE TITLE
forcefield tree not correctly found

### ISSUE THREAD

**Issue by Djurredejong** [mnmelo/molmaker#1]
Hi Manel,
It seems to me the code to find the "standard GMX forcefield tree" is broken. On line 148:
`self.gmxlibdir = re.sub("/bin/grompp", "/share/gromacs/top", self.grompp)`
self.grommp is not an string (but list). You could fix it by changing it to:
`self.gmxlibdir = re.sub("/bin/gmx", "/share/gromacs/top", self.gmx)`

**Comment by mnmelo** [comment]
Thanks for bringing this up, Djurre. I made a comment on your PR (#2).

**Comment by Djurredejong** [comment]
Yeah, next time I go all smarty pants on your code I should actually check if it does work. When I now did I found the next two lines also needed fixing.

One more thing I noticed (I know I should open a new issue for this, but it really isn't that important and I shouldn't expect such fool-proof broad support anyway): I also got an error due to the "-nt","1" option set for "gmx mdrun" because my mdrun wasn't thread-mpi compiled.
